tests/features/bootstrap/P2Context.php

```diff
@@ -75,13 +62,40 @@ public function iHaveSearchedFor($arg1)
      */
     public function iAddTheSearchToFollowedSearches()
     {
-        $followed_searches_id = $this->iReadTheListIdForListName('user-searches');
-        $found = $this->ding2Context->minkContext->getSession()->getPage()
-            ->find('css', 'a[href^="/dinglist/attach/search_query/' . $followed_searches_id . '"]');
-        if (!$found) {
-            throw new \Exception("Couldn't find button to add search to list.");
+        $followed_searches_id = $this->getListId('follow search');
+        $this->moreDropdownSelect('/dinglist/attach/search_query/'. $followed_searches_id, "Couldn't find button to add search to list");
+    }
+
+    /**
+     * Select an item in a "More.." dropdown.
+     *
+     * @param string $link
+     *   The link the item should point to.
+     * @param string $errorMessage
+     *   Exception message if the link could not be found.
+     */
+    public function moreDropdownSelect($link, $errorMessage)
+    {
+        $page = $this->ding2Context->minkContext->getSession()->getPage();
+        $button = $page->find('css', '.ding-list-add-button a');
+        if (!$button) {
+            throw new \Exception("Couldn't find more button");
         }
-        $found->click();
+
+        try {
+            // Mouseover the button to trigger the dropdown. Can't click an
+            // invisible link in a real browser.
+            $button->mouseOver();
```

Comment:
Nogle gange er det nødvendigt at scrolle ned til knappen først. Det er nok en god ide at gøre hver gang.
$this->ding2Context->minkContext->getSession()
            ->evaluateScript('jQuery(document).scrollTo(".ding-list-add-button a");');